polaris-react/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx

```diff
@@ -79,7 +79,7 @@ export function ContextualSaveBar({
 
   const saveActionMarkup = saveAction && (
     <Button
-      primary
+      primarySuccess
```

Comment:
I don't see an issue attached and am likely missing context, but should this be beta flagged?

Reply:
No beta flag needed, the non-experimental primary and primarySuccess are the same (that's why there were no UI test changes)